src/cmdlinetest/test_snakepit.t

```diff
@@ -88,6 +91,29 @@
    '--debug': True,
    '--force': True,
    '--help': False,
+   '--output': None,
    '--version': False,
    '<file>': 'snakepit.yaml'}
   Writing output to: 'moto.spec'
+
+# test specified output is an existent directory
```

Comment:
The negative case testing is good, but you should test the positive ones too. i.e.:

```
$snakepit snakepit.yaml --output foo.yaml
$ ls foo.yaml
foo.yaml
```

and 

```
$ mkdir bar
$ snakepit snakepit.yaml --output bar/foo.yaml
$ ls bar/
foo.yaml

```
Or something similar.